src/utils/filter-products.js

```diff
@@ -0,0 +1,38 @@
+export const filterProducts = (
+  productList,
+  { fiction, scifi, novel, selfhelp, biography, comic }
+) => {
+  let filteredProduct = [];
+  if (fiction === true)
```

Comment:
don't need to write if(fiction===true) instead we can write if(fiction) it will also works. you can change this in all your code.

Reply:
sure, will change.